userpanel/modules/helpdesk/templates/setup.html

```diff
@@ -103,13 +103,9 @@ <h1>{$layout.pagetitle}</h1>
 								{trans("Default categories:")}
 							</td>
 							<td class="nobr">
+								<select class='lms-ui-advanced-select' name='lms_categories[]' multiple>
```

Comment:
A tu dlaczego już wartości atrybutów w apostrofach?

Reply:
W przypadku szablonów nie gra to roli ale w przypadku PHP już tak (szybkość kodu). Jak chcesz zebym zostawił?